scripts/release/homebrew/templates/heroku.rb

```diff
@@ -11,7 +11,7 @@ class Heroku < Formula
 
   def install
     inreplace "bin/heroku", /^CLIENT_HOME=/, "export HEROKU_OCLIF_CLIENT_HOME=#{lib/"client"}\nCLIENT_HOME="
-    inreplace "bin/heroku", "\"$DIR/node\"", Formula["heroku-node"].opt_share/"node"
+    inreplace "bin/heroku", "\"$DIR/node\"", Formula["heroku-node"].bin/"node"
```

Comment:
Thanks for catching this! this needs to be `opt_bin` in order for it to work. I tested it locally using `brew reinstall --build-from-source Formula/heroku.rb` 

```suggestion
    inreplace "bin/heroku", "\"$DIR/node\"", Formula["heroku-node"].opt_bin/"node"
```

Reply:
Ah yes of course!